Controls/motor_powers.py

```diff
@@ -0,0 +1,63 @@
+import math
```

Comment:
This file might not be necessary at all based on the current locomotion plan, so we can get rid of it if we need. It could possibly be handy to work off of if we do end up building some kind of model in the future.

Reply:
Yeah i'd keep it and just label it unused